File: app/src/main/java/com/example/myfakequotesapp/Room/Repository.java
package com.example.myfakequotesapp.Room;

import android.app.Application;
import android.widget.Toast;

public class Repository {
    //static QuoteRoomDatabase database;
    static RepositoryTasks database;
    static Application app;

    static public void initDataBase(Application application){
        app = application;
        if(database == null){
            database = new QuoteRepository(application);
        }
    }
    static public RepositoryTasks getDataBase(){
        if(database == null) {
            Toast.makeText(app,"No db", Toast.LENGTH_LONG).show();
        }
        return database;
    }
}

File: app/src/main/java/com/example/myfakequotesapp/Room/Repository.kt
package com.example.myfakequotesapp.Room

import android.widget.Toast
import android.app.Application

object Repository {
    //static QuoteRoomDatabase database;
    var database: RepositoryTasks? = null
    var app: Application? = null
    fun initDataBase(application: Application) {
        app = application
        if (database == null) {
            database = QuoteRepository(application)
        }
    }

    val getDataBase: RepositoryTasks?
        get() {
            if (database == null) {
                Toast.makeText(app, "No db", Toast.LENGTH_LONG).show()
            }
            return database
        }
}